src/index.html

```diff
@@ -33,12 +33,16 @@ <h2>Add your score</h2>
                 type="text"
                 class="input-name"
                 placeholder="Your Name"
+                minlength="4"
+                maxlength="15"
                 required
               />
               <input
                 type="number"
                 class="input-score"
                 placeholder="Your Score"
+                min="0"
+                max="9000000000"
```

Comment:
## Hello Cynthia :wave: 
## :recycle:  [OPTIONAL CHANGES]  :recycle: 
you did a great job with adding some form validations. 
I however feel that a value of ```9000000000``` is too large and un reasonable  for a competition. I would recommend a value like 1000.

Reply:
Yeah, I know the number is to much. It was just to test but I will modify the number

Reply:
Okay that's great